exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebServer.java

```diff
@@ -96,6 +107,23 @@
 
   private Server embeddedJetty;
 
+  private static File tmpJavaScriptDir;
```

Comment:
Can we store it as local variable?

Reply:
Being a single instance, I thought a static would be preferable.